styles/mixins/tooltip.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function () {
+  return function (mixin, color) {
+    var color = color || 'rgba(17,17,17,.9)';
```

Comment:
any chance we can get direction passed in here? if no direction passed, then loop through and set everything. That way we can be consistent with our other mixins